lib/seattleflu/id3c/cli/command/etl/__init__.py

```diff
@@ -137,6 +137,33 @@ def standardize_race(race):
     return list(map(standardize_race, races))
 
 
+def ethnicity(redcap_record: dict) -> Optional[str]:
```

Comment:
@kschwabefry Good thought, I changed it to string parameter. In looking into ethnicity further, I realized that it should be using a coding system (`valueCoding`) instead of `valueString`, so I modified the ethnicity function to return a dict with the appropriate FHIR ethnicity coding. 

Looking at our other redcap_det ETLs, `ethnicity` typically gets ingested as a boolean value (most likely because that's how it's defined in REDCap) but I think those should all be updated use coded values like I have here. That will also require updates in shipping views, so if it's better to keep it a boolean for now (for consistency) I can update my PR to remove the coding.